src/explore-education-statistics-admin/src/services/releaseAncillaryFileService.ts

```diff
@@ -46,20 +46,16 @@ const releaseAncillaryFileService = {
   uploadAncillaryFile(
     releaseId: string,
     request: UploadAncillaryFileRequest,
-  ): Promise<AncillaryFile> {
+  ): Promise<void> {
```

Comment:
This isn't particularly good RESTful API design. Just because the frontend currently doesn't do anything with the result doesn't mean that the API shouldn't return anything.

Reply:
Agreed - Changed it back to returning the AncillaryFile :)